Update hunting.md (#94749)
Azure/azure-content: articles/sentinel/hunting.md

1. Modify the query and select **Create**.

**To modify an existing custom query**

1. From the table, select the hunting query that you wish to modify. Note that only queries that from a custom content source can be edited. Other content sources have to be edited at that source.

> 1. Select the ellipsis (...) in the line of the query you want to modify, and select **Edit query**.

Reviewer: ```suggestion

1. Select the ellipsis (...) in the line of the query you want to modify, and select **Edit query**.
```

> 1. Modify the **Custom query** field with the updated query. You can also modify the entity mapping and techniques as per the "**To create a new query**" section og this documentation

Reviewer: ```suggestion

1. Modify the **Custom query** field with the updated query. You can also modify the entity mapping and techniques as explained in the "**To create a new query**" section of this documentation.
```
